pipelines/pipeline/main.tf

```
@@ -53,12 +53,35 @@ resource "aws_codebuild_project" "plan" {
 
 resource "aws_codebuild_webhook" "plan" {
   project_name  = aws_codebuild_project.plan.name
-  branch_filter = "^(?!master|develop).*$"
+
+  filter_group {
+    filter {
+      type = "EVENT"
+      pattern = "PUSH, PULL_REQUEST_CREATED, PULL_REQUEST_UPDATED"
+    }
+
+    filter {
+      type = "HEAD_REF"
+      pattern = "^refs/heads/(?:master|develop)$"
```

@chrismshelton Do you think we need to escape those forward slashes?
```suggestion
      pattern = "^refs\/heads\/(?:master|develop)$"
```

---

@tom-fleming Good catch - I would think so, but they don't escape them here: https://docs.aws.amazon.com/codebuild/latest/userguide/github-webhook.html#github-webhook-events-sdk